src/python/pants/notes/1.30.x.rst

```diff
@@ -0,0 +1,205 @@
+1.30.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.30.x`` ``stable`` series.
+
+See https://pants.readme.io/v1.29/docs/release-notes-1-30 for an overview of the changes in this release.
```

Comment:
So, you'll want to fork the docs to create a v1.30 link. Don't mark it as "main" yet. The release process probably needs to be updated to clarify this.

```suggestion
See https://pants.readme.io/v1.30/docs/release-notes-1-30 for an overview of the changes in this release.
```